bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/form/FormStructureParserImpl.java

```diff
@@ -118,6 +120,7 @@ public String getFormDefinition() {
         FormContainer formContainer = resource.adaptTo(FormContainer.class);
         try {
             ObjectMapper mapper = new ObjectMapper();
+            mapper.registerModule(new SimpleModule().addSerializer(String.class, new AbstractBaseImpl.EncodeHTMLSerializer()));
```

Comment:
lets keep the serializer outside AbstractBase.

Reply:
Done